cmd/codescanning.go

```diff
@@ -13,21 +15,26 @@ import (
 
 func init() {
 	codeScanningCmd.PersistentFlags().StringVarP(&Organization, "organization", "o", "", "specify Organisation to implement code scanning")
-	codeScanningCmd.MarkPersistentFlagRequired("organization")
-	codeScanningCmd.PersistentFlags().StringVarP(&WorkflowFile, "workflow-file", "f", "", "specify the path to the code scanning workflow file")
-	codeScanningCmd.MarkPersistentFlagRequired("workflow-file")
-	codeScanningCmd.PersistentFlags().StringVarP(&LogFile, "log-file", "l", "", "specify the path where the log file will be saved")
-	codeScanningCmd.MarkPersistentFlagRequired("log-file")
-
+	codeScanningCmd.PersistentFlags().StringVarP(&WorkflowFile, "workflow", "w", "", "specify the path to the code scanning workflow file")
+	codeScanningCmd.MarkPersistentFlagRequired("workflow")
+	codeScanningCmd.PersistentFlags().StringVarP(&LogFile, "log", "l", "code-scanning.out", "specify the path where the log file will be saved")
```

Comment:
I think it's a great idea to create default logs.

Nitpick -  The name `code-scanning.out` can be confusing if the user doesn't know that this is the default name as it can be confused with the github code scanning run.  Maybe it should be `gh-add-files.log` 

Any reason it's `.out` versus `.log`. Usually for log files I use `.log` because it's just obvious what it is whilst `.out` can be any output.

Reply:
> Maybe it should be gh-add-files.log

Great idea! I'll change that

> Any reason it's .out versus .log

Pure laziness 🤣 I just used that because it was already in the `.gitignore` file but I just added a `.log` entry there now so we good!